### ISSUE TITLE
Solid Aria npm package

### ISSUE THREAD

**Issue by davedbase** [lawrencecchen/solid-aria#1]
Hello! I'm contacting on behalf of SolidJS Community as a Core Solid team member. We're currently working with @fabien-ml to setup a new solid-aria project to port react-aria. It's quite the commitment and job. I noticed that you have control of the solid-aria name on npm and was wondering if you'd be willing to donate it to our effort: https://github.com/solidjs-community. SolidJS Community is a community run holdings org for a lot of key/critical Solid projects.

We'd love for you to be involved in this critical effort and hoped you'd be willing to work with us on it or at least be willing to transfer solid-aria for our ability to publish to.

Let me know!

David

**Comment by lawrencecchen** [comment]
Yeah, that sounds like a good idea. Would love to get involved and donate the npm package name, lmk details.

**Comment by davedbase** [comment]
That's so wonderful. @fabien-ml and I are putting together a project and a team on SolidJS Community to tackle this. React Aria is what we're trying to port and it's a BIG effort. We'd absolutely love your input and support as a joint effort.

In regards to the npm.org project did you want to add myself and fabien to the publishing list on the package? My handle is just davedbase. :-)

If you're interested in working with us I can add you to the Aria Team in SolidJS Community. I plan to setup a Discord channel for us to organize as well.

**Comment by fabien-ml** [comment]
For me it's `fabienml` on npm. Maybe @davedbase can be owner on the npm org too.

**Comment by lawrencecchen** [comment]
Added, please add me to the discord channel when it's ready. My handle is pacifistbunny#9855

**Comment by fabien-ml** [comment]
@lawrencecchen can you create the solid-aria organisation on npm ?

**Comment by davedbase** [comment]
@fabien-ml I already control that org :-)
